src/main/scala/com/gu/mobiledatalakealerts/AwsCredentials.scala

```diff
@@ -1,18 +1,16 @@
 package com.gu.mobiledatalakealerts
 
-import com.amazonaws.auth.{AWSCredentialsProviderChain, EnvironmentVariableCredentialsProvider}
+import com.amazonaws.auth.{ AWSCredentialsProviderChain, EnvironmentVariableCredentialsProvider }
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
 
 object AwsCredentials {
 
   val athenaCredentials = new AWSCredentialsProviderChain(
     new ProfileCredentialsProvider("ophan"),
-    new EnvironmentVariableCredentialsProvider()
-  )
+    new EnvironmentVariableCredentialsProvider())
 
   val notificationCredentials = new AWSCredentialsProviderChain(
-    new ProfileCredentialsProvider("mobile"),
-    new EnvironmentVariableCredentialsProvider()
-  )
+    new ProfileCredentialsProvider("developerPlayground"),
```

Comment:
These credentials allow us to push to Anghammarad's SNS topic. Since we want this service to be reusable, I tried to choose a profile/account which all developers should have access to (instead of `mobile` which should be more restrictive).